shaky/src/main/res/layout/shaky_single_row.xml

```diff
@@ -18,24 +18,23 @@
 			  xmlns:tools="http://schemas.android.com/tools"
               android:layout_width="match_parent"
               android:layout_height="wrap_content"
-              android:background="?attr/selectableItemBackground"
               android:clickable="true"
               android:focusable="true"
               android:gravity="center_vertical"
               android:minHeight="?android:attr/listPreferredItemHeight"
               android:orientation="horizontal"
-              android:padding="@dimen/button_padding">
+              android:padding="@dimen/shaky_button_padding">
 
     <ImageView
         android:id="@+id/shaky_row_icon"
         android:layout_width="wrap_content"
         android:layout_height="match_parent"
-        android:layout_margin="@dimen/icon_padding"
+        android:layout_margin="@dimen/shaky_icon_padding"
         android:adjustViewBounds="true"
         android:gravity="center_vertical"
         android:scaleType="centerInside"
 		tools:ignore="ContentDescription"
-        android:src="@drawable/img_lightbulb_56dp"/>
+        android:src="@drawable/shaky_img_lightbulb_56dp"/>
```

Comment:
We should let users pick out their icons as well, by converting this imageview + textview combo into a textview with a start drawable. Not so sure these icons will work well on dark mode, and it'll be nice to be able to customize them

Reply:
Since this is used in a RecyclerView the icons have to get set at runtime, this icon is just a default placeholder. Since our layout is more complex that a textview with start drawable (we use a LinearLayout to arrange the title and subtitle) I don't think that will work